### PR TITLE
Function for rejection dictionary

### PR DESCRIPTION
**Pull request by jasmainak**
@agramfort does this do the job for you?

### CODE REVIEW

File: autoreject/autoreject.py [hunk 1]
```diff
@@ -140,6 +148,39 @@ def fit(self, X, y=None):
...
+        The epochs for which to estimate the epochs dictionary
+
+    Returns
+    -------
+    reject : dict
+        The rejection dictionary with keys 'meg', 'eeg' and 'eog'
+    """
+    reject = dict()
+    X = epochs.get_data()
+    picks = channel_indices_by_type(epochs.info)
```
Comment (jasmainak) [inline autoreject/autoreject.py]: I need to add a line to exclude the bad channels when picking ...

File: autoreject/autoreject.py [hunk 2]
```diff
@@ -140,6 +148,39 @@ def fit(self, X, y=None):
         return self
 
 
+def get_rejection_threshold(epochs):
+    """Class to compute global rejection thresholds.
```
Comment (agramfort) [inline autoreject/autoreject.py]: Compute global rejection thresholds.

File: autoreject/autoreject.py [hunk 3]
```diff
@@ -140,6 +148,39 @@ def fit(self, X, y=None):
...
+
+    Parameters
+    ----------
+    epochs : mne.Epochs object
+        The epochs for which to estimate the epochs dictionary
```
Comment (agramfort) [inline autoreject/autoreject.py]: for -> from

**Comment by jasmainak** [comment]
Thanks for the review @agramfort

File: examples/plot_estimate_global_reject.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+                       eog=True, include=include, exclude='bads')
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
+                    picks=picks, baseline=(None, 0),
+                    reject=None, verbose=False, detrend=1)
+
+###############################################################################
+# Now we get the rejection dictionary
+
+from autoreject import get_rejection_threshold  # noqa
+reject = get_rejection_threshold(epochs)
```
Comment (agramfort) [inline examples/plot_estimate_global_reject.py]: what value of reject do you get?

File: examples/plot_estimate_global_reject.py [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+print('The rejection dictionary is %s' % reject)
```
Comment (agramfort) [inline examples/plot_estimate_global_reject.py]: I would then do:

epochs.drop_bad(reject=reject)
epochs.average().plot()

File: examples/plot_estimate_global_reject.py (hunk 4 above)
Comment (jasmainak) [inline examples/plot_estimate_global_reject.py]: 'eog': 6.5597820032270958e-05, 'eeg': 6.6741382744238379e-05, 'grad': 3.9468601180690814e-11, 'mag': 1.123921842576036e-12

**Comment by agramfort** [comment]
can you update the example?

**Comment by jasmainak** [comment]
@agramfort how [about now](http://autoreject.github.io/auto_examples/plot_estimate_global_reject.html)

**Comment by agramfort** [comment]
looks good

**Pull request merged**